### PR TITLE
Kokoro android artifact

### PR DESCRIPTION
**Pull request by ericgribkoff**

### CODE REVIEW

File: buildscripts/kokoro/linux_artifacts.sh [hunk 1]
```diff
@@ -18,3 +18,16 @@ docker build -t protoc-artifacts /tmp/protobuf/protobuf-92898e9e9cb2f1c006fcc509
 docker build -t grpc-java-releasing "$GRPC_JAVA_DIR"/buildscripts/grpc-java-releasing
 
 "$GRPC_JAVA_DIR"/buildscripts/run_in_docker.sh /grpc-java/buildscripts/build_artifacts_in_docker.sh
+
+LOCAL_MVN_TEMP=$(mktemp -d)
+pushd "$GRPC_JAVA_DIR/android"
+../gradlew uploadArchives \
+  --include-build .. \
```
Comment (zpencer) [inline buildscripts/kokoro/linux_artifacts.sh]: nit: $GRPC_JAVA_DIR feels a bit clearer than `..`

Comment (ericgribkoff) [reply buildscripts/kokoro/linux_artifacts.sh]: Done

File: buildscripts/kokoro/linux_artifacts.sh [hunk 2]
```diff
@@ -18,3 +18,18 @@ docker build -t protoc-artifacts /tmp/protobuf/protobuf-92898e9e9cb2f1c006fcc509
...
+
+# grpc-android requires the Android SDK, so build outside of Docker and
+# use --include-build for its grpc-core dependency
+LOCAL_MVN_TEMP=$(mktemp -d)
+pushd "$GRPC_JAVA_DIR/android"
+../gradlew uploadArchives \
+  --include-build "$GRPC_JAVA_DIR" \
+  -Dorg.gradle.parallel=false \
+  -PskipCodegen=true \
+  -PrepositoryDir=$LOCAL_MVN_TEMP
```
Comment (zpencer) [inline buildscripts/kokoro/linux_artifacts.sh]: quote the LOCAL_MVN_TEMP

Comment (ericgribkoff) [reply buildscripts/kokoro/linux_artifacts.sh]: Done

**Pull request merged**